### ISSUE TITLE
TypeError: expected str, bytes, or os.PathLike object, not NoneType

### ISSUE THREAD

**Issue by CDAGaming** [achillesrasquinha/pipupgrade#84]
#### What's the bug all about?

This issue has only just began occurring today, where the only recent edits to my environment involved the following commands, which succeeded at the time.

```
#!/bin/bash
# Upgrade Auto Script v1
#
# Linux Upgrade Data
sudo apt update
sudo apt upgrade -y
# PIP Upgrade Data
pipupgrade -y --ignore distro-info --ignore python-apt
# Brew Upgrade Data
brew update
brew upgrade
# Await further input before closure
echo
read -rsn1 -p "Press any key to continue . . ."
echo
```

I am unsure what could have occurred to cause pipupgrade to stop working but I'm open to any sort of suggestions (distro-info and python-apt are ignored in the script, since PopOS manages both those and Pipupgrade detects them as a false-positive update)

#### What command led to this issue?

Any pipupgrade command leads to the same error.

#### Logs

https://gist.github.com/CDAGaming/9d3cd811dcc834dadee9eeea548ff171

```
cdagaming@pop-os:~$ neofetch
             /////////////                cdagaming@pop-os 
         /////////////////////            ---------------- 
      ///////*767////////////////         OS: Pop!_OS 20.04 LTS x86_64 
    //////7676767676*//////////////       Host: Predator PH315-51 V1.26 
   /////76767//7676767//////////////      Kernel: 5.8.0-7630-generic 
  /////767676///*76767///////////////     Uptime: 12 mins 
 ///////767676///76767.///7676*///////    Packages: 2884 (dpkg), 1 (rpm), 48 (b 
/////////767676//76767///767676////////   Shell: bash 5.0.17 
//////////76767676767////76767/////////   Resolution: 1920x1080 
///////////76767676//////7676//////////   DE: GNOME 
////////////,7676,///////767///////////   WM: Mutter 
/////////////*7676///////76////////////   WM Theme: Pop 
///////////////7676////////////////////   Theme: Pop-dark [GTK2/3] 
 ///////////////7676///767////////////    Icons: Pop [GTK2/3] 
  //////////////////////'////////////     Terminal: gnome-terminal 
   //////.7676767676767676767,//////      CPU: Intel i7-8750H (12) @ 4.100GHz 
    /////767676767676767676767/////       GPU: Intel UHD Graphics 630 
      ///////////////////////////         GPU: NVIDIA GeForce GTX 1060 Mobile 
         /////////////////////            Memory: 4146MiB / 31951MiB 
             /////////////
                                                                  
                                                                  
```

**Comment by duduche59** [comment]
Same issue 
OS : Linux Mint 20

**Comment by zombiepigdragon** [comment]
Reproduced on Arch Linux today. Can be fixed in the _very_ short term by downgrading to 1.7.4 with `pip install 'pipupgrade==1.7.4'`, though obviously using it will bring back the broken version 1.8.0.

**Comment by achillesrasquinha** [comment]
Fixed and Released `1.8.1`.

**Comment by CDAGaming** [comment]
Closing as Fixed, but not fully (Forwarded to #85
